src/plugins/condition/components/inspector/StyleEditor.vue

```diff
@@ -78,6 +78,7 @@
     <!-- Save Styles -->
     <toolbar-button
       v-if="canSaveStyle"
+      ref="saveStyleButton"
```

Comment:
Added these refs so we can access them in the unit tests instead of using `$children` (which has been removed)